ipaplatform/debian/tasks.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+#
+
+"""
+This module contains default Debian-specific implementations of system tasks.
+"""
+
+from ipaplatform.paths import paths
+from ipaplatform.base.tasks import *
```

Comment:
Please don't use wildcard imports, QuantifiedCode is having a bad day if you do.